omnigibson/object_states/robot_related_states.py

```diff
@@ -50,13 +51,10 @@ def _get_value(self, obj):
 #         return not body_ids.isdisjoint(robot.states[ObjectsInFOVOfRobot].get_value())
 
 
-# class ObjectsInFOVOfRobot(AbsoluteObjectState):
-#     def _get_value(self):
-#         # Pass the FOV through the instance-to-body ID mapping.
-#         seg = self.simulator.renderer.render_single_robot_camera(self.obj, modes="ins_seg")[0][:, :, 0]
-#         seg = np.round(seg * MAX_INSTANCE_COUNT).astype(int)
-#         body_ids = self.simulator.renderer.get_pb_ids_for_instance_ids(seg)
-
-#         # Pixels that don't contain an object are marked -1 but we don't want to include that
-#         # as a body ID.
-#         return set(np.unique(body_ids)) - {-1}
\ No newline at end of file
+class ObjectsInFOVOfRobot(AbsoluteObjectState):
+    def _get_value(self):
+        for sensor in self.robot.sensors.values():
+            if isinstance(sensor, VisionSensor):
+                _, info = sensor.get_obs()
+                return [prim_path for prim_path in info['seg_instance'].values() if prim_path.startswith('/')]
```

Comment:
A couple things:

- Is there ever a case where a prim path will not start with a `/`?
- Shouldn't we iterate over all vision sensors? so something like:

```python
paths = set()
for sensor in ...:
    new_paths = {set(sensor.values()}
   paths.update(new_paths)
```

?

Reply:
1. I was trying to get rid of these two labels: `'BACKGROUND', 'UNLABELLED'`. I can also just explicitly take these two out instead of the using `/` as a filter. 
2. Yes, that's a good catch! Forgot to include all of them.

Reply:
Ahhh I see. Yeah let's explicitly catch those two edge cases